.github/workflows/build-and-publish.yaml

```diff
@@ -50,3 +50,10 @@ jobs:
           tox -e build,twinecheck
       - name: Publish package distributions to PyPI
         uses: pypa/gh-action-pypi-publish@release/v1
+      - name: Free disk space
```

Comment:
Added to incorrect github workflow, this is in build-and-publish which is used for building the pypi wheel and publishing it but the error is in the [image workflow](https://github.com/foundation-model-stack/fms-hf-tuning/blob/main/.github/workflows/image.yaml)